pages/components/Header.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function Header(props) {
```

Comment:
before we merge tho let's remove unused code (unless there's plans to use this in the future, if so, id leave a `// TODO: add description here` comment